samples/magic/az_ml_magic.py

```diff
@@ -0,0 +1,300 @@
+from __future__ import print_function
+from IPython.core.magic import (Magics, magics_class, line_magic, cell_magic)
+from IPython.utils.ipstruct import Struct
+import os
+import argparse
+import shlex
+import re
+
+from cStringIO import StringIO
+import sys
+
+
+class Capturing(list):
+    def __enter__(self):
+        self._stdout = sys.stdout
+        sys.stdout = self._stringio = StringIO()
+        return self
+
+    def __exit__(self, *args):
+        self.extend(self._stringio.getvalue().splitlines())
+        del self._stringio  # free up some memory
+        sys.stdout = self._stdout
+
+
+@magics_class
+class AMLHelpers(Magics):
+    @staticmethod
+    def print_and_update_env(k, v):
+        os.environ[k] = v
+        print(' os.environ["{}"]="{}"'.format(k, v))
+        return 'export {}={}'.format(k, v)
```

Comment:
So this export statement will only work in a bash-like shell, right? Can we do something like what the CLI does to print these statements based on the OS?